apicast/conf.d/apicast.conf

```diff
@@ -26,6 +26,10 @@ location = /threescale_authrep {
   proxy_set_header  X-3scale-Version "$version";
   proxy_set_header  Connection "";
 
+  # Enable the option to receive the rejection reason in the
+  # '3scale-rejection-reason' header of the response.
+  proxy_set_header  3scale-options "rejection_reason_header=1";
```

Comment:
Could we make this read from a variable? So it can be customized?

Reply:
I missed this comment. Will address in a different PR.